master/lustrefactory.py

```diff
@@ -72,6 +83,54 @@ def buildCommand(props):
 
     return args
 
+def getTarballFactory(gerrit_repo, **kwargs):
```

Comment:
"create" would probably be better than "get" as a prefix for this function name.